metaflow/mflog/__init__.py

```diff
@@ -41,21 +41,23 @@
     " }" % TASK_LOG_SOURCE
 )
 
-BASH_SAVE_LOGS_ARGS = ["python", "-m", "metaflow.mflog.save_logs"]
-BASH_SAVE_LOGS = " ".join(BASH_SAVE_LOGS_ARGS)
+
+def _get_bash_capture_log(default_python_executable="python"):
+    bash_save_log_args = [default_python_executable, "-m", "metaflow.mflog.save_logs"]
+    return bash_save_log_args, " ".join(bash_save_log_args)
```

Comment:
we could just return the array part and have the user do the join. I don't have a strong preference but the [0], and [1] on use are a bit weird.